tests/infrastructure/numa/test_basic_numa.py

```
@@ -42,10 +43,9 @@ def test_numa_cpu_allocation(self, created_vm_cx1_instancetype):
             numa_nodes=get_numa_node_cpu_dict(vm=created_vm_cx1_instancetype),
         )
 
+    @pytest.mark.rwx_default_storage
     @pytest.mark.polarion("CNV-12368")
-    def test_live_migrate_numa_vm(
-        self, skip_access_mode_rwo_scope_class, created_vm_cx1_instancetype, migrated_numa_cx1_vm
-    ):
+    def test_live_migrate_numa_vm(self, created_vm_cx1_instancetype, migrated_numa_cx1_vm):
```

Ruff (0.14.10)

Unused method argument: `migrated_numa_cx1_vm`

(ARG002)